### PR TITLE
PMM-9168 remove the review workflow

### PR DESCRIPTION
**Pull request by atymchuk**
[PMM-9168](https://jira.percona.com/browse/PMM-9168) (optional, if ticket reported)

- [ ] Links to other linked pull requests (optional).
---
- [ ] Tests passed.
- [x] Fix conflicts with target branch.
- [x] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

When all checks have passed and code is ready for the review, bot should add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forums](https://forums.percona.com) and [Discord](https://discord.gg/mQEyGPkNbR).

### CODE REVIEW

File: .github/workflows/ready-for-review.yml [hunk 1]
```diff
@@ -1,15 +0,0 @@
-name: "Assign Reviewers"
```
Comment (BupycHuk) [inline .github/workflows/ready-for-review.yml:1]: Why did you remove it?

Comment (atymchuk) [reply .github/workflows/ready-for-review.yml:1]: Please take a look at the ticket.

**Review by denisok — APPROVED** [review]

File: .github/workflows/ready-for-review.yml (hunk 1 above)
Comment (denisok) [reply .github/workflows/ready-for-review.yml:1]: it doesn't work: https://github.com/rowi1de/auto-assign-review-teams/issues/17#issuecomment-967261575
